python/test/unit/language/test_standard.py

```diff
@@ -0,0 +1,181 @@
+import triton
+import pytest
+import torch
+import triton.language as tl
+import numpy as np
+
+from test_core import _test_binary, int_dtypes, uint_dtypes, float_dtypes, is_hip, numpy_random, check_type_supported, to_triton, to_numpy, RandomState, patch_kernel
```

Comment:
@ThomasRaoux There are still shared utilities among test_core.py and test_standard.py. 
Would it be better to create another file and import?

Reply:
You mean having a shared utilities file? Yeah that would work but to be honest I think it is fine to pull helpers from test_core.